File: ApiSuvesaPos/Negocio/Modelo/cxp/ordencompra/detalle_ordencompra.vb
Namespace Modelo
    Public Class detalle_ordencompra
        Public Property Id As Integer
        Public Property Orden As Integer
        Public Property Codigo As Integer
        Public Property Descripcion As String
        Public Property CostoUnitario As Decimal
        Public Property Cantidad As Decimal
        Public Property TotalCompra As Decimal
        Public Property Porc_Descuento As Decimal
        Public Property Descuento As Decimal
        Public Property Porc_Impuesto As Decimal
        Public Property impuesto As Decimal
        Public Property OtrosCargos As Decimal
        Public Property Monto_Flete As Decimal
        Public Property Costo As Decimal
        Public Property Gravado As Decimal
        Public Property Exento As Decimal
        Public Property Vendidos As Decimal
        Public Property Exist_Actual As Decimal
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxp/ordencompra/detalle_ordencompra.cs
namespace Modelo
{
    public class detalle_ordencompra
    {
        public int Id { get; set; }
        public int Orden { get; set; }
        public int Codigo { get; set; }
        public string Descripcion { get; set; }
        public decimal CostoUnitario { get; set; }
        public decimal Cantidad { get; set; }
        public decimal TotalCompra { get; set; }
        public decimal Porc_Descuento { get; set; }
        public decimal Descuento { get; set; }
        public decimal Porc_Impuesto { get; set; }
        public decimal impuesto { get; set; }
        public decimal OtrosCargos { get; set; }
        public decimal Monto_Flete { get; set; }
        public decimal Costo { get; set; }
        public decimal Gravado { get; set; }
        public decimal Exento { get; set; }
        public decimal Vendidos { get; set; }
        public decimal Exist_Actual { get; set; }
    }
}
